### PR TITLE
Remove opencensus jaeger exporter

### PR DESCRIPTION
**Pull request by gzhao408**
Part of #133, subtask of #132  

Remove jaeger exporter from checkoutservice, shippingservice, frontend, and adservice because it isn't necessary to export to anything other than GCP and OpenCensus is being deprecated.

### CODE REVIEW

File: src/checkoutservice/main.go [hunk 1]
```diff
@@ -192,6 +168,10 @@ func (cs *checkoutService) Check(ctx context.Context, req *healthpb.HealthCheckR
 	return &healthpb.HealthCheckResponse{Status: healthpb.HealthCheckResponse_SERVING}, nil
 }
 
+func (cs *checkoutService) Watch(req *healthpb.HealthCheckRequest, srv healthpb.Health_WatchServer) error {
+	return nil
+}
```
Comment (simonz130) [inline src/checkoutservice/main.go:173]: Is this related to removing jaeger?

Comment (gzhao408) [reply src/checkoutservice/main.go:173]: sorry no, i was getting a build error about it so I added it in there and it worked as a bandaid but is probably not the issue. I'm going to rebase and try again without it, will open a separate pr if it's actually necessary

Comment (gzhao408) [reply src/checkoutservice/main.go]: I was clearly confused 😅  just removed all of them

File: src/checkoutservice/main.go [hunk 2]
```diff
@@ -154,7 +131,6 @@ func initStackDriverTracing() {
 }
 
 func initTracing() {
-	initJaegerTracing()
```
Comment (Daniel-Sanche) [inline src/checkoutservice/main.go:157]: nit: this function should probably be removed if it contains a single line now

File: src/shippingservice/main.go [hunk 3]
```diff
@@ -185,7 +162,6 @@ func initStackDriverTracing() {
 }
 
 func initTracing() {
-	initJaegerTracing()
```
Comment (Daniel-Sanche) [inline src/shippingservice/main.go:188]: same as above

File: src/checkoutservice/main.go (hunk 2 above)
Comment (gzhao408) [reply src/checkoutservice/main.go:157]: done and done
